src/Keboola/StorageApi/ProcessPolyfill.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Keboola\StorageApi;
+
+use Symfony\Component\Process\Process;
+
+final class ProcessPolyfill
+{
+    /**
+     * @param string $cmdString
+     * @return Process
+     */
+    public static function createProcess($cmdString)
```

Comment:
Případně to šlo udělat přes https://github.com/Ocramius/PackageVersions ;)

Reply:
Jo to šlo, ale je to zbytečná závislost. Taky bys musel používat hodně historicků verzi aby to fungovalo s PHP 5.6 ocramius sa s podporů starých verzí moc nesere :D